### PR TITLE
Scrum_76_delete_pressentations

### PR DESCRIPTION
**Pull request by Jasonn5**
This pull request introduces a comprehensive suite of test cases for meeting deletion functionality within the EspoCRM API. The focus is to ensure robustness and reliability of the delete operation under various scenarios. The tests are designed to validate proper API behavior, including successful deletion, handling of invalid requests, and correct response codes.

Setup Method:
Function: setup_create_meeting
Purpose: Creates a meeting before each test to ensure there is a valid meeting available for deletion.
Details:

- Generates headers using valid credentials.
- Creates a meeting and yields its ID along with the headers for use in the test.
Test Cases Implemented:

Test Method: test_delete_existing_meeting =>  Expected Result: 200 OK
Test Method: test_delete_non_existent_meeting =>Expected Result: 404 Not Found
Test Method: test_delete_meeting_without_auth => Expected Result: 401 Unauthorized
Test Method: test_delete_meeting_returns_ok => Expected Result: 200 OK
Test Method: test_deleted_meeting_not_recoverable => Expected Result: 404 Not Found
Test Method: test_delete_meeting_invalid_id => Expected Result: 404 Not Found
Test Method: test_delete_meeting_unauthorized_user => Expected Result: 401 Unauthorized
Test Method: test_delete_meeting_expected_headers =>xpected Result: Valid headers in response

Test Method: test_delete_meeting_confirmation_response_body
Expected Result: Response body 'true'
Ensures: Proper response body format.
image

### CODE REVIEW

**Review by ErickaIsa — APPROVED** [review]
No issues observed with the submitted code.

File: api_endpoints/api_request.py
Comment (AlisonGuzman2006) [inline api_endpoints/api_request.py:1]: The changes are reviewed and accepted

File: api_endpoints/meeting_endpoints.py
Comment (AlisonGuzman2006) [inline api_endpoints/meeting_endpoints.py:1]: The addition to the code is reviewed and accepted.

File: tests/presentations/conftest.py
Comment (AlisonGuzman2006) [inline tests/presentations/conftest.py:1]: The addition to the code is reviewed and accepted

File: tests/presentations/test_delete_meeting.py
Comment (AlisonGuzman2006) [inline tests/presentations/test_delete_meeting.py:1]: The addition to the code is reviewed and accepted

**Review by AlisonGuzman2006 — APPROVED** [review]
The changes and additions to the code have been reviewed and accepted

**Pull request merged**